it/google-cloud-platform/src/main/java/org/apache/beam/it/gcp/TemplateTestBase.java

```diff
@@ -90,11 +90,13 @@ public abstract class TemplateTestBase {
   public TestRule watcher =
       new TestWatcher() {
         protected void starting(Description description) {
-          LOG.info(
-              "Starting integration test {}.{}",
-              description.getClassName(),
-              description.getMethodName());
           testName = description.getMethodName();
+          // In case of parameterization the testName can contain subscript like testName[paramName]
+          // Converting testName from testName[paramName] to testNameParamName since it is used to
+          // create many resources and it cannot contain special characters.
```

Comment:
Why did this not fail earlier? What has changed?

Reply:
We never had parameterized tests in ITs in this repo. So, the testName never contained subscripts earlier.

Reply:
As mentioned in the comment the subscript is actually parameter name.